### PR TITLE
Update eCommerce copy

### PR DESCRIPTION
**Pull request by obenland**
#### Changes proposed in this Pull Request

* Adds features for eCommerce plan

#### Testing instructions

* Try adding an eCommerce plan

### CODE REVIEW

**Comment by obenland** [comment]
image
@sara Should we add the "All Business Plan Features" feature as the third item here?

**Comment by obenland** [comment]
image
image

**Comment by sararosso** [comment]
a few comments: 

> https://github.com/Automattic/wp-calypso/pull/28602#issuecomment-439176410
on this view, we should be seeing the tabbed design view - @michaeldcain perhaps you have some input here? 

The taglines for the different plans needs updating: 
Business: The plan for small businesses
eCommerce: The plan for online stores

> Should we add the "All Business Features" feature as the third item here?
Yes, please!

For the descriptor on the plans page let's make sure to capitalize Title Case like the other plans, so it should be: 
> Best for Online Stores

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -1183,7 +1202,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_UPLOAD_THEMES_PLUGINS ]: {
 		getSlug: () => FEATURE_UPLOAD_THEMES_PLUGINS,
-		getTitle: () => i18n.translate( 'Upload themes and plugins' ),
+		getTitle: () => i18n.translate( 'Upload Themes and Plugins' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 22 translations.

**Comment by obenland** [comment]
screen shot 2018-11-15 at 7 55 59 pm

Plans page when I leave "Sell products or collect payments" unchecked

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -353,7 +359,7 @@ const getPlanBusinessDetails = () => ( {
 	getAudience: () => i18n.translate( 'Best for small businesses' ),
 	getBlogAudience: () => i18n.translate( 'Best for brands' ),
 	getPortfolioAudience: () => i18n.translate( 'Best for small businesses' ),
-	getStoreAudience: () => i18n.translate( 'The plan for stores and small businesses' ),
+	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :disappointed: 20 existing translations will be lost with this change.

**Comment by akirk** [comment]
Having the same text for title and description with just a difference in casing is really not useful and will result in double translation. Can't we leave the description empty if we have none?

**Comment by sararosso** [comment]
@akirk  there's a new updated PR here https://github.com/Automattic/wp-calypso/pull/28945

**Pull request merged**